tensorzero-core/src/db/clickhouse/select_queries.rs

```diff
@@ -27,23 +27,23 @@ impl SelectQueries for ClickHouseConnectionInfo {
         // NOTE: this filter pattern will likely include some extra data since the current period is likely incomplete.
         let (time_grouping, time_filter) = match time_window {
             TimeWindow::Hour => (
-                "toStartOfHour(minute)",
+                "toStartOfHour(minute)".to_string(),
                 format!("minute >= (SELECT max(toStartOfHour(minute)) FROM ModelProviderStatistics) - INTERVAL {max_periods} HOUR"),
             ),
             TimeWindow::Day => (
-                "toStartOfDay(minute)",
+                "toStartOfDay(minute)".to_string(),
                 format!("minute >= (SELECT max(toStartOfDay(minute)) FROM ModelProviderStatistics) - INTERVAL {max_periods} DAY"),
             ),
             TimeWindow::Week => (
-                "toStartOfWeek(minute)",
+                "toStartOfWeek(minute)".to_string(),
                 format!("minute >= (SELECT max(toStartOfWeek(minute)) FROM ModelProviderStatistics) - INTERVAL {max_periods} WEEK"),
             ),
             TimeWindow::Month => (
-                "toStartOfMonth(minute)",
+                "toStartOfMonth(minute)".to_string(),
                 format!("minute >= (SELECT max(toStartOfMonth(minute)) FROM ModelProviderStatistics) - INTERVAL {max_periods} MONTH"),
             ),
             TimeWindow::Cumulative => (
-                "toDateTime('1970-01-01 00:00:00')",
+                "toDateTime('1970-01-01 00:00:00')".to_string(),
```

Comment:
why? if you don't need to do this then please remove as the hard string type can be statically allocated